_src/static/sass/components/_buttons.scss

```diff
@@ -0,0 +1,389 @@
+// #PRIMARY
+// ----------------------------
+.btn-primary {
+    background: palette(primary, mid-dark);
+    border-color: palette(primary, mid-dark);
+    color: palette(grayscale, white);
+
+    // STATE: hover and focus
+    &:hover,
+    &.is-hovered,
+    &:focus,
+    &.is-focused {
+        background: palette(primary, base);
+        border-color: palette(primary, base);
+    }
+
+    // STATE: is pressed or active
+    &:active,
+    &.is-pressed,
+    &.is-active {
+        background: palette(primary, mid-dark);
+        border-color: palette(primary, mid-dark);
+    }
+
+    // STATE: is disabled
+    &:disabled,
+    &.is-disabled {
+        background: palette(grayscale, x-light);
+        border-color: palette(grayscale, x-light);
+        color: palette(grayscale, base);
+    }
+}
+
+
+// ----------------------------
+// #SECONDARY
+// ----------------------------
+.btn-secondary {
+    background: $transparent;
+    border-color: $transparent;
+    color: palette(primary, base);
+
+    // STATE: hover and focus
+    &:hover,
+    &.is-hovered,
+    &:focus,
+    &.is-focused {
+        border-color: palette(grayscale-cool, trans);
+    }
+
+    // STATE: is pressed or active
+    &:active,
+    &.is-pressed,
+    &.is-active {
+        border-color: palette(primary, dark);
+        color: palette(primary, mid-dark);
+    }
+
+    // STATE: is disabled
+    &:disabled,
+    &.is-disabled {
+        border-color: palette(grayscale, x-light);
+        color: palette(grayscale, mid-light);
+    }
+}
+
+
+// ----------------------------
+// #SECONDARY-ALT
+// ----------------------------
+.btn-secondary-alt {
+    background: $transparent;
+    border-color: $transparent;
+    color: palette(grayscale-cool, base);
+
+    // STATE: hover and focus
+    &:hover,
+    &.is-hovered,
+    &:focus,
+    &.is-focused {
+        background: palette(grayscale-cool, base);
+        border-color: palette(grayscale-cool, base);
+        color: palette(grayscale, white);
+    }
+
+    // STATE: is pressed or active
+    &:active,
+    &.is-pressed,
+    &.is-active {
+        background: palette(primary, dark);
+        border-color: palette(primary, dark);
+    }
+
+    // STATE: is disabled
+    &:disabled,
+    &.is-disabled {
+        border-color: palette(grayscale, x-light);
+        color: palette(grayscale, mid-light);
+    }
+}
+
+
+// ----------------------------
+// #LINK
+// ----------------------------
+.btn-link {
+    background: $transparent;
+    border-color: $transparent;
+    color: palette(primary, mid-dark);
+
+    // STATE: hover and focus
+    &:hover,
+    &.is-hovered,
+    &:focus,
+    &.is-focused {
+        color: palette(primary, base);
+        text-decoration: underline;
+    }
+
+    // STATE: is pressed or active
+    &:active,
+    &.is-pressed,
+    &.is-active {
+        color: palette(primary, dark);
+    }
+
+    // STATE: is disabled
+    &:disabled,
+    &.is-disabled {
+        display: none; // :)
```

Comment:
Doesn't matter to me. I was basically saying that links don't really have disabled states, so if we're using an inactive link (or a link-button), we should just not show it in the UI. Like, how do you make a link look inactive without making it look like text? You could fade it back, but then you risk running into a11y issues for contrast.